internal/elasticsearch/index/data_stream_lifecycle/update.go

```diff
@@ -0,0 +1,53 @@
+package data_stream_lifecycle
+
+import (
+	"context"
+
+	"github.com/elastic/terraform-provider-elasticstack/internal/clients"
+	"github.com/elastic/terraform-provider-elasticstack/internal/clients/elasticsearch"
+	"github.com/elastic/terraform-provider-elasticstack/internal/utils"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func (r *Resource) Update(ctx context.Context, req resource.UpdateRequest, resp *resource.UpdateResponse) {
```

Comment:
This looks identical to the `Create` method. Can we pull this out to a common `write` (or some other name if you prefer) method and re-use it in both `Create` and `Update`?

Reply:
OK, I did the same as here:

internal/fleet/integration/create.go
internal/fleet/integration/update.go